gui/mozregui/global_prefs.py

```diff
@@ -24,6 +25,8 @@ def save_prefs(options):
     settings.update({
         'persist': options['persist'] or '',
         'http-timeout': options['http_timeout'],
+        'persist-size-limit': int(abs(float(options.persist_size_limit))
```

Comment:
Looks good, but *options.persist_size_limit* won't work! You should use options['persist_size_limit'].
Also the float conversion is not required here I think since we did that in get_prefs().